File: uhealthy/src/main/java/com/fmu/uhealthy/domain/Clinic.java
package com.fmu.uhealthy.domain;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import javax.persistence.*;

@Data
@NoArgsConstructor
@AllArgsConstructor
@Entity(name="clinic")
public class Clinic {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;

    @Column(name = "name")
    private String name;

    @Column(name = "company_name")
    private String companyName;

    @Column(name = "address")
    private String address;
}

File: front/lib/model/clinic.dart
class Clinic {
  final int? id;
  final String? name;
  final String? companyName;
  final String? address;

  Clinic(this.id, this.name, this.companyName, this.address);

  static convertClinic(clinic) {
    return Clinic(clinic['id'], clinic['name'], clinic['company_name'],
        clinic['address']);
  }

  Map<String, dynamic> toJson() {
    final _data = <String, dynamic>{};
    _data['id'] = id;
    _data['name'] = name;
    _data['companyName'] = companyName;
    _data['address'] = address;
    return _data;
  }
}
